algebird-core/src/main/scala/com/twitter/algebird/CountMinSketch.scala

```diff
@@ -964,7 +972,7 @@ class TopNCMSMonoid[K: Ordering](cms: CMS[K], heavyHittersN: Int = 100) extends
   /**
    * Creates a sketch out of a single item.
    */
-  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms, params)
+  def create(item: K): TopCMS[K] = TopCMSItem[K](item, cms + item, params)
```

Comment:
I added two tests for each Top-% CMS and Top-N CMS that test for this bug.  Note that because the counting table has private access I use "identity" inner products (= the inner product of a CMS instance with itself) to verify correct behavior instead of e.g. verifying that the L1-norm of each `w` vector is 1.